facebook-instant-articles.php

```diff
@@ -373,9 +373,11 @@ function inject_url_claiming_meta_tag() {
 	 * @since 4.0
 	 */
 	function inject_ia_markup_meta_tag() {
-		$post = get_post();
-
-		// If there's no current post, return
+		if ( !is_singular() )
+			return;
+		
+		$post = get_post( get_queried_object_id() );
```

Comment:
Under what conditions would this now give a different result than it did before? Can you add an integration test to cover it?